.github/workflows/zcashd.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: stable
+          toolchain: nightly
```

Comment:
we shouldn't target `nightly`, as it's routinely broken

Reply:
unless we _really_ need some unstable feature?

Reply:
Well, `--format json` is currently locked behind the nightly compiler. So, unless we want to complicate things further, there's not a whole lot we can do.